src/qml/Mysthea/Pages/CardsReferencePage.qml

```diff
@@ -21,6 +22,20 @@ Page {
         id: typeProxyModel
     }
 
+    // We created this connection to show translated content text of
+    // combobox when changed application language.
```

Comment:
Would be better if you say why this has to be done. 
It's clear from the code that this is changing the two `ComboBox` text, but not why.